products/health-care/appointments/va-online-scheduling/research/july-2020-user-research/vaosr-june-research-plan.md

```diff
@@ -1,50 +1,86 @@
 # Research Plan for - VAOS Team on VA.gov, VAOSR, June & July 2020
+
 As a designer I need to create a research plan to share with my team.	(This should be filled out and maintained by the designer on the team & can be cross-linked and referenced in any issues.) 	
+
 ## Goals	
+
 1. What product & team are you doing this research for?	
+   VAOSR on VA.gov
 2. Background: Briefly, what is the background on this product? What would a new person on the team need to know about this product? 	
-3. Research questions: What question(s) do you hope to be able to answer after completing this research? 	
+   With VAOSR MVP in prod and live since March 2020, it's time to perform another user research and testing with Veterans.
+   This research should focus on areas in the experience that we know (through analytics and user feedback) are confusing or need iteration. Additionally, we want to collect general feedback and requests from Veterans for new VAOSR considerations in the future.
+3. Research questions: What question(s) do you hope to be able to answer after completing this research? 
+   1. Do Veterans understand the login and tool instructions?
+   2. Do Veterans understand what they see on the appointments list? Is the appointments list useful to veterans? 
+   3. Does the "past appointments" list (new feature) meet veterans' needs?
+   4. Are veterans able to cancel appointments? Are veterans having issues when trying to make cancellations? Is the experience of cancelling an appointment what they expect?
+   5. What's impacting "type of care" dropoff?  Does "type of care" offerings meet veterans expectation for online scheduling?
```

Comment:
changed

I agree it will come out organically. I tend to err on the side of not asking that question directly, but asking around it. People often don't really know what they want, but they can speak to what isn't working, what is working, and what they need.